openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/EntityLineage.component.tsx

```diff
@@ -1654,9 +1653,7 @@ const EntityLineageComponent: FunctionComponent<EntityLineageProp> = ({
                 onOptionSelect={handleOptionSelect}
               />
             )}
-            {isEditMode && (
-              <Background gap={12} size={1} variant={BackgroundVariant.Lines} />
-            )}
+            <Background gap={12} size={1} />
```

Comment:
Don't we need isEditMode condition here?

Reply:
Rupesh asked to keep background in both edit and non edit mode.